saleor/graphql/order/dataloaders.py

```diff
@@ -51,6 +52,17 @@ def batch_load(self, keys):
         return [line_map.get(order_id, []) for order_id in keys]
 
 
+class PaymentsByOrderIdLoader(DataLoader):
+    context_key = "payments_by_order"
+
+    def batch_load(self, keys):
+        payments = Payment.objects.filter(order_id__in=keys).order_by("pk")
+        payment_map = defaultdict(list)
+        for line in payments.iterator():
```

Comment:
Why `line` ?

Reply:
copy pasted it a bit too much, thanks for the catch!